data/Gaussian/basicGaussian16/water_ccd.log

```diff
@@ -1,10 +1,11 @@
- Entering Gaussian System, Link 0=g16_main
```

Comment:
Rather than update the unit test files, can you instead add these files to https://github.com/cclib/cclib-data/, then add tests to https://github.com/cclib/cclib/blob/master/test/regression.py that check for `ccenergies` for that file?  If you need help understanding the test format in that file, let us  know.

Reply:
The test file has been merged into cclib-data which should allow the regression test to run once added.